### PR TITLE
Document SSH settings

### PR DESCRIPTION
**Pull request by qwerty287**
was added back in **2.2.0** https://github.com/woodpecker-ci/plugin-git/pull/75

### CODE REVIEW

File: docs.md [hunk 1]
```diff
@@ -56,8 +56,11 @@ clone:
...
 | `sha`                     | $CI_COMMIT_SHA                      | git commit hash to retrieve (use `sha: ''` to clone the `ref`)                                                                          |
 | `ref`                     | $CI_COMMIT_REF                      | Set the git reference to retrieve (use `ref: refs/heads/a_branch` and `sha: ''` to retrieve the head commit from the "a_branch" branch) |
 | `path`                    | $CI_WORKSPACE                       | Set destination path to clone to                                                                                                        |
+| `use_ssh`                 | `false`                             | Clone using SSH                                                                                                                         |
+| `use_ssh`                 | `false`                             | SSH key for SSH clone                                                                                                                   |
```
Comment (lafriks) [inline docs.md:64]: This seems wrong name

File: docs.md [hunk 2]
```diff
@@ -56,8 +56,11 @@ clone:
...
 | `sha`                     | $CI_COMMIT_SHA                      | git commit hash to retrieve (use `sha: ''` to clone the `ref`)                                                                          |
 | `ref`                     | $CI_COMMIT_REF                      | Set the git reference to retrieve (use `ref: refs/heads/a_branch` and `sha: ''` to retrieve the head commit from the "a_branch" branch) |
 | `path`                    | $CI_WORKSPACE                       | Set destination path to clone to                                                                                                        |
+| `use_ssh`                 | `false`                             | Clone using SSH                                                                                                                         |
+| `ssh_key`                 | `false`                             | SSH key for SSH clone                                                                                                                   |
```
Comment (lafriks) [inline docs.md:64]: Default value also probably is not false

Comment (6543) [inline docs.md:64]:
```suggestion
| `ssh_key`                 | _none_                             | SSH key for SSH clone                                                                                                                   |
```

File: docs.md [hunk 3]
```diff
@@ -56,8 +56,11 @@ clone:
...
 | `sha`                     | $CI_COMMIT_SHA                      | git commit hash to retrieve (use `sha: ''` to clone the `ref`)                                                                          |
 | `ref`                     | $CI_COMMIT_REF                      | Set the git reference to retrieve (use `ref: refs/heads/a_branch` and `sha: ''` to retrieve the head commit from the "a_branch" branch) |
 | `path`                    | $CI_WORKSPACE                       | Set destination path to clone to                                                                                                        |
+| `use_ssh`                 | `false`                             | Clone using SSH                                                                                                                         |
+| `ssh_key`                 | _none_                             | SSH key for SSH clone                                                                                                                   |
```
Comment (6543) [inline docs.md:64]:
```suggestion
| `ssh_key`                 | _none_                              | SSH key for SSH clone                                                                                                                   |
```

**Review by 6543 — APPROVED** [review]

**Pull request merged**